pkgs/node/src/chain.zig

```diff
@@ -495,3 +662,118 @@ test "printSlot output demonstration" {
     try std.testing.expect(beam_chain.forkChoice.protoArray.nodes.items.len == mock_chain.blocks.len);
     try std.testing.expect(beam_chain.registered_validator_ids.len == 3);
 }
+
+test "save and load block" {
+    var arena_allocator = std.heap.ArenaAllocator.init(std.testing.allocator);
```

Comment:
Is there memory leak when change the allocator to `testing.allocator`, arena allocator seems hide memory leak issue so that we should use `testing.allocator` and `gpa`(multithread test) in the tests